docs/conf.py

```diff
@@ -25,7 +25,7 @@
 import packaging.version
 
 project = 'OpenPathSampling CLI'
-copyright = '2019-2020, David W.H. Swenson'
+copyright = '2019-2022, David W.H. Swenson'
```

Comment:
```suggestion
copyright = '2019-2022, David W.H. Swenson and contributors'
```

I'm taking credit as the main author (especially since this copyright notification is technically for the docs) but this should acknowledge the existence of other contributors.